apps/src/templates/codeReviewGroups/CodeReviewGroupsManager.jsx

```diff
@@ -24,6 +24,28 @@ export default function CodeReviewGroupsManager({initialGroups}) {
   const getAssignedGroups = () =>
     groups.filter(group => group.droppableId !== DROPPABLE_ID_UNASSIGNED);
 
+  const onGroupNameUpdate = (droppableId, newName) => {
+    const updatedGroup = {...getGroup(droppableId), ...{name: newName}};
```

Comment:
nit: I think you can simplify this to `{...getGroup(droppableId), name: newName}`